plugins/modules/web_infrastructure/jira.py

```diff
@@ -81,18 +83,28 @@
     required: false
     description:
      - The comment text to add.
+     - Note that JIRA may not allow changing field values on specific transitions or states.
 
   status:
     type: str
     required: false
     description:
-     - The desired status; only relevant for the transition operation.
+     - Only used when C(operation) is I(transition), and a bit of aa misnomer, it actually refers to the transition name.
 
   assignee:
     type: str
     required: false
     description:
-     - Sets the assignee on create or transition operations. Note not all transitions will allow this.
+     - Sets the the assignee when C(operation) is I(create), I(transition) or I(edit).
+     - Recent versions of JIRA no longer accept an user name as an user identifier. In that case, use C(account_id) instead.
```

Comment:
```suggestion
     - Recent versions of JIRA no longer accept a user name as a user identifier. In that case, use C(account_id) instead.
```

Reply:
It is a rusty memory but I believe the article before vowels should be `an` instead of `a`. :grin:

Reply:
The problem is that it depends on the pronounciation. The `u` in `user` beings with a consonant, `/j/` or `/y/` in IPA (see https://en.wiktionary.org/wiki/user#Pronunciation and https://en.wiktionary.org/wiki/Appendix:English_pronunciation#Consonants), so it's `a user` and not `an user` :)

Reply:
oh my.. you really **did** your home work on that one :-)